packages/camera/camera/example/ios/RunnerTests/ThreadSafeEventChannelTests.m

```diff
@@ -0,0 +1,63 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera;
+@import XCTest;
+#import <OCMock/OCMock.h>
+
+@interface ThreadSafeEventChannelTests : XCTestCase
+@property(nonatomic, strong) FLTThreadSafeEventChannel *channel;
+@property(nonatomic, strong) XCTestExpectation *mainThreadExpectation;
+@property(nonatomic, strong) XCTestExpectation *mainThreadCompletionExpectation;
```

Comment:
changed from ivars to properties

Reply:
Please don't make expectations a stateful part of the fixture; that makes tests much harder to understand.

I don't see a need for these properties or `setUp` to exist at all. It would be preferable for all of the code to be in each test.

Reply:
@stuartmorgan Very interesting. I did a bit search and found out that we generally prefer declarative coding style in unit test, where readability is more important than DRY principle. Today I learned.

Reply:
I think I still want the `setUp` function to setup the test subject (the wrapper and the mock channel dependency).